### PR TITLE
Add unit test for ClusterContextManagerCoordinator.completeUnitShowProcessList(final ShowProcessListUnitCompleteEvent event)

### PR DESCRIPTION
**Pull request by iisheng**
Fixes #16968.

### CODE REVIEW

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/ClusterContextManagerCoordinatorTest.java [hunk 1]
```diff
@@ -351,6 +356,39 @@ public void assertRenewInstanceOnlineEvent() {
         assertThat(((LinkedList<ComputeNodeInstance>) contextManager.getInstanceContext().getComputeNodeInstances()).get(1).getInstanceDefinition(), is(instanceDefinition1));
     }
     
+    @Test
+    public void assertCompleteUnitShowProcessList() {
+        final String showProcessListId = "show_process_list_id_1";
+        final ShowProcessListSimpleLock simpleLock = new ShowProcessListSimpleLock();
+        ShowProcessListManager.getInstance().getLocks().put(showProcessListId, simpleLock);
+
```
Comment (terrymanu) [inline shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/ClusterContextManagerCoordinatorTest.java:364]: Please remove unnecessary blank line

Comment (terrymanu) [inline shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/ClusterContextManagerCoordinatorTest.java:364]: Please remove unnecessary blank line

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/ClusterContextManagerCoordinatorTest.java [hunk 2]
```diff
@@ -351,6 +356,39 @@ public void assertRenewInstanceOnlineEvent() {
         assertThat(((LinkedList<ComputeNodeInstance>) contextManager.getInstanceContext().getComputeNodeInstances()).get(1).getInstanceDefinition(), is(instanceDefinition1));
     }
     
+    @Test
+    public void assertCompleteUnitShowProcessList() {
+        final String showProcessListId = "show_process_list_id_1";
```
Comment (terrymanu) [inline shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/ClusterContextManagerCoordinatorTest.java:361]: What is the mean of `show_process_list_id_1`,  how about change it to `foo_process_id`?

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/ClusterContextManagerCoordinatorTest.java [hunk 3]
```diff
@@ -351,6 +356,39 @@ public void assertRenewInstanceOnlineEvent() {
...
+        final ShowProcessListSimpleLock simpleLock = new ShowProcessListSimpleLock();
```
[the remark this answers is not in the document]
Comment (iisheng) [reply shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/ClusterContextManagerCoordinatorTest.java]: done

**Review by terrymanu — APPROVED** [review]

**Pull request merged**